tests/storage/blockvolume_test.py

```diff
@@ -72,8 +72,7 @@ class TestBlockVolumeSize(VdsmTestCase):
         # Sparse, cow, capacity config.volume_utilization_chunk_mb - 1,
         # No initial size.
         # Expected 1024 MiB allocated (config.volume_utilization_chunk_mb)
-        [(sc.SPARSE_VOL, sc.COW_FORMAT, (CONFIG.getint(
-            "irs", "volume_utilization_chunk_mb") - 1) * MiB, None), GiB],
+        [(sc.SPARSE_VOL, sc.COW_FORMAT, (CHUNK_SIZE_MB - 1) * MiB, None), GiB],
```

Comment:
Not sure this is better - it is shorter code but the current code is about the actual vdsm apis used, so it is more clear what is going on. Adding a constant hide things.

Reply:
For a single occurrence is ok, for multiple occurrences it made the permutations more difficult to read.

I think the constant declared in consecutive lines with the `CONFIG`, makes it clear that the permutations use the direct size, and the test shall obtain the same size from the `CONFIG` structure that is monkeypatched.

Which btw made me realize that there is a typo in the `CONFIG`, `volume_utilizzation_precent` is wrong.